pages/00-Home.md

```diff
@@ -1,87 +1,67 @@
 ---
 layout: page
-title: Foundations | Hydraecosystem.org
+title: Tools | Hydraecosystem.org
 permalink: /00-Home
 ---
 
-# Foundations
+# Generics
 
-## Generics
-Besides all the other tools in the ecosystem, **hydrus** (lowercase) is the basic module from which API designers/engineers/developers can build up their own automated infrastructure to leverage smart clients. hydrus is the server-side tool that allows REST data to be published as Hydra-aware data. A **Hydra network** is made up of one or more hydrus instances that receives requests from **Hydra smart clients** (or agents, see [`hydra-python-agent`](https://github.com/HTTP-APIs/hydra-python-agent) repository). The interaction of hydrus and Hydra smart clients is how the Hydra ecosystem realizes the Client-Server pattern.
+## Hydrus
 
-hydrus provides a full-stack architecture that makes it the basic building block for a machine that is a node in a wider network of Web APIs. Generic components of the network are:
-* on hydrus, a REST interface
-* on hydrus, a Hydra-powered server, with the possibilities of being self-deployable and accepting on-the-fly configuration changes
-* on the smart client, a Hydra-powered generic client which can connect to any Hydra-based APIs without hard-coding its behaviour
+**Hydrus** is a set of Python based tools for easier and efficient creation of Hypermedia driven REST-APIs. Hydrus utilizes the power of Linked Data to create a powerful REST APIs to serve data. Hydrus uses the Hydra(W3C) standard for creation and documentation of it’s APIs. It is a project created from scratch with minimal requirements, so that anyone can use it in their project without any difficulties. Lets's simplify things a bit and try to find what is hydrus and how does it work actually? So it's a python based library, which enables us to create (smart) APIs which can be interpreted by machines. And for that to happen we just need to supply hydrus a API doc (created with the help of Hydra vocabulary) to it. So now you might be wondering that how does it all actually happens? Let's clear the mist around it and discuss it's working. In order to leverage benefit of hydrus library apart from dependent python libraries we don't need much apart from an API doc. Actually hydrus exptects hydraDoc object, but when we pass a API Doc to it. It internally uses the Doc Maker tool create a HydraDoc object for it. After receiving hydraDoc object, hydrus parses the data into `Classes`, `Properties`, `Collections`. Which are then utilized to create smart APIs.  
```

Comment:
Change the hydrus definition to: "Hydrus is a generic server stack that is used  for easier and efficient creation of Hypermedia driven REST-APIs with the use of Hydra.

Also change the voice to third person. And avoid using questions in between.

Reply:
Also you just need to introduce the project here. No need to explain the working. Remove all content about hydraDoc objects and parsing.

A homepage should just contain small explanations of the tools. Not the working.